### DISCUSSION TITLE
How would i make the goal post fall down when i finish a stage

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#100] [Q&A] [answered]
How would i make the goal post fall down when i finish a stage

**Comment by DimensionWarped** [comment, marked as answer]
This is something for you to figure out on your own. Generally speaking, you'd use some kind of trigger at the end of a boss fight to kick this off... whether that's sending a signal and having it caught by something or just a simple function call to another script. It's not complicated, but if you want to use the engine proficiently at all you need to figure out how to think through this kind of thing on your own.

This discussion board isn't here to help you figure out how to do every little thing from start to finish and you should really heed Techokami's words from your last thread. I've already warned you multiple times about posting questions like this without having specific problems that you are looking to figure out... and to clarify, when I say specific problems, I'm talking about things that you have put substantial thought into and can create a significant document describing *how* you have already tried to implement it and why you think there might be some actual engine limitation holding you back. If you keep making posts like "How do I make a triforce?", Techokami is going to ban you soon.

This is your final warning with regard to that. I'm not going to ask Techokami to hold back next time. You already know how to contact me privately, so just ask me if your questions are appropriate for the discussion board from now on.
